matrix/common/etl/transformers/cell_expression.py

```diff
@@ -88,13 +88,20 @@ def _parse_ss2_bundle(self, bundle_dir):
 
         genes_detected = sum((1 for k in genes_values.values() if k["Count"] > 0))
 
+        file_uuid = [f for f in json.load(open(bundle_manifest_path))["files"]
+                     if f["name"].endswith(".genes.results")][0]["uuid"]
+        file_version = [f for f in json.load(open(bundle_manifest_path))["files"]
+                        if f["name"].endswith(".genes.results")][0]["version"]
```

Comment:
Can we have/is the choice of `.genes.results`, `matrix.mtx` and `.zattrs` as the identifying file documented somewhere more public than the code here for developer purposes?

Reply:
No not currently, I'll write something up we can put on the repo and maybe on the browser.